GMG/Scenes/Home/HomeIntent.swift

```diff
@@ -41,6 +41,10 @@ final class HomeIntent: HomeIntentProtocol {
 
     func selectScore(_ score: Score?) {
         model?.setSelectedScore(score)
+
+        guard let score else { return }
+
+        setupPlayer(for: score)
```

Comment:
score null 체크를 하는거면 model?.setSelectedScore(score) 위에 선언하는게 맞을 듯 싶습니다.


```suggestion
guard let score else { return }

model?.setSelectedScore(score)

setupPlayer(for: score)
```